README.md

```diff
@@ -28,6 +28,13 @@ Import the TypeScript types:
 import * as schema from 'custom-elements-manifest/schema';
 ```
 
+## Schema Versioning
+
+The schema has a `schemaVersion` field in the top-level object to facilitate
+evolution of the schema. The current schema version is `0.1.0`.
```

Comment:
```suggestion
evolution of the schema.
```

Suggest removing that, since it'll become out of sync and isn't necessary to make the point I think.

Reply:
I want to move this to a better location eventually, but I think we need to document what to put in the `schemaVersion` field right now. Otherwise there's no guidance at all, and the field becomes meaningless. I'll look up how other JSON schema projects manage versions, but I think we need to keep this for now.